tests/conftest.py

```diff
@@ -125,6 +140,23 @@ def setup():
     cfg.main.chain_id = 1337
     ganache_endpoint = RPCEndpoint(1337, url="http://localhost:8545")
     cfg.endpoints.endpoints.append(ganache_endpoint)
+    mainnet_endpoint = cfg.endpoints.find(chain_id=1)
+    if mainnet_endpoint and mainnet_endpoint[0].url.endswith("{INFURA_API_KEY}"):
+        mainnet_endpoint[0].url = os.getenv("MAINNET_URL")
+
+    w3 = Web3(Web3.HTTPProvider("http://127.0.0.1:8545"))
+    token = w3.eth.contract(address=token_contract_info.address[1], abi=token_contract_info.get_abi(1))
+    transfer = token.get_function_by_name("transfer")
+    # transfer 100 TRB to the disputer account
+    token_txn = transfer("0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1", int(100e18)).buildTransaction(
+        {
+            "gas": 500000,
+            "gasPrice": w3.eth.gas_price,
+            "nonce": w3.eth.get_transaction_count("0x39E419bA25196794B595B2a595Ea8E527ddC9856"),
+            "from": "0x39E419bA25196794B595B2a595Ea8E527ddC9856",
```

Comment:
What account is this? Maybe add a brief inline comment explaining what account / what used for